packages/react/src/components/TextField/TextField.types.ts

```diff
@@ -168,6 +168,12 @@ export interface ITextFieldProps extends React.AllHTMLAttributes<HTMLInputElemen
    */
   readOnly?: boolean;
 
+  /**
+   * If true, the text field is invalid. Will be auto-determined by errorMessage unless set.
+   * @defaultvalue false
+   */
+  invalid?: boolean;
```

Comment:
yup, this was added for that reason, because previously you couldn't set the invalid semantics & styles without an error message. Would you prefer the `aria-invalid` check? I'd be worried that could break existing customers, since I imagine there are people who already pass in `aria-invalid` directly, and have their own styling fix

Reply:
I could go either way. I can see how this case is somewhat distinct, but there's just something about having a third or fourth way to trigger similar functionality... 😬